tslint.json

```diff
@@ -11,7 +11,7 @@
         "no-exec-script": true,
         "no-function-constructor-with-string-args": true,
         "no-http-string": [
-            true,
+            false, // changed

```

Comment:
Is this turning the rule off?  I'd rather leave it on but disable per line.

Reply:
Okay, can do.